Dockerfile.release

```diff
@@ -0,0 +1,13 @@
+# This Dockerfile is used to build container image for production workloads.
+# It relies on the binaries produced by `ignite_release` and `ignite_release_extract_binaries` make targets.
+FROM alpine:3.19
+ARG TARGETARCH
+
+# Use `1025` G/UID so users can switch between this and `heighliner` image without a need to chown the files.
+RUN addgroup --gid 1025 -S pocket && adduser --uid 1025 -S pocket -G pocket
+
+COPY --chown=pocket:pocket release_binaries/poktroll_linux_$TARGETARCH /bin/poktrolld
+
+USER pocket
+
+ENTRYPOINT ["poktrolld"]
```

Comment:
Why is the poktrolld binary the entrpoint?

Reply:
> Why is the poktrolld binary the entrpoint?

glad you asked. 

It allows to easy set up a shell alias to use `poktrolld` from docker without installing the binary on the machine. I'll showcase this in the docs.

Reply:
#PUC

```
# poktrolld is the entrypoint to allow easily setting up a shell alias to use poktrolld from docker without installing the binary on the machine. I'll showcase this in the docs.
ENTRYPOINT ["poktrolld"]
```

Reply:
@okdas ☝️